ansible/inventory.yml

```diff
@@ -3,3 +3,6 @@ servers:
   hosts:
```

Comment:
The line I'm trying to comment (3) is just cut out of the diff, so GitHub won't let me comment it. If we're adding a new inventory "staging", we should probably rename the existing one to "production"